io_u.c

```diff
@@ -866,21 +867,42 @@ static int fill_io_u(struct thread_data *td, struct io_u *io_u)
 		goto out;
 
 	/*
-	 * See if it's time to switch to a new zone
+	 * if file is zoned, zone_range is set to positive value always.
```

Comment:
"When file is zoned zone_range is always positive"

Reply:
Done